<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1013</width>
    <height>449</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="VkAuthView" name="authView" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="userId_label">
          <property name="text">
           <string>user_id:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="userId_lineEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="copyUserId_toolButton">
          <property name="text">
           <string>Copy</string>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="accessToken_label">
          <property name="text">
           <string>access_token:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="accessToken_lineEdit"/>
        </item>
        <item>
         <widget class="QToolButton" name="copyAccessToken_toolButton">
          <property name="text">
           <string>Copy</string>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="expiresIn_label">
          <property name="text">
           <string>expires_in:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="expiresIn_lineEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="copyExpiresIn_toolButton">
          <property name="text">
           <string>Copy</string>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="apiVersion_label">
        <property name="text">
         <string>API version:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="apiVersion_comboBox"/>
      </item>
      <item>
       <widget class="QLabel" name="appId_label">
        <property name="text">
         <string>AppID:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="appId_spinBox">
        <property name="maximum">
         <number>999999999</number>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="permissions_label">
        <property name="text">
         <string>Permissions:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="permissions_listWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="auth_pushButton">
        <property name="text">
         <string>Start authorization</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1013</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="quit_action"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="about_action"/>
    <addaction name="aboutQt_action"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="quit_action">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="about_action">
   <property name="text">
    <string>&amp;About...</string>
   </property>
  </action>
  <action name="aboutQt_action">
   <property name="text">
    <string>About &amp;Qt...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>VkAuthView</class>
   <extends>QWidget</extends>
   <header location="global">vkauthview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>apiVersion_comboBox</tabstop>
  <tabstop>appId_spinBox</tabstop>
  <tabstop>permissions_listWidget</tabstop>
  <tabstop>auth_pushButton</tabstop>
  <tabstop>userId_lineEdit</tabstop>
  <tabstop>copyUserId_toolButton</tabstop>
  <tabstop>accessToken_lineEdit</tabstop>
  <tabstop>copyAccessToken_toolButton</tabstop>
  <tabstop>expiresIn_lineEdit</tabstop>
  <tabstop>copyExpiresIn_toolButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
